src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -54,7 +54,24 @@ const getForgotPasswordLink = (appState) => {
   }
 };
 
+const goBackLink = (appState) => {
+
+  if (appState.hasRemediationObject(RemediationForms.ENROLL_AUTHENTICATOR)) {
```

Comment:
shall check for `SELECT_AUTHENTICATOR_ENROLL` ?

Reply:
I think she is checking were the request is coming from

Reply:
+1 we should check for `SELECT_AUTHENTICATOR_ENROLL` form here as that is the form we are going back to.

Reply:
my bad yeah.

Reply:
Yea, I thought we were supposed to check where we are coming from. This makes sense. Made changes.